### PR TITLE
PLAT-2003 deployment guide

### PR DESCRIPTION
**Pull request by cassandrabailey293**
Create a deployment guide on how to deploy OpenTDF to an AWS cluster. This is the first iteration of the deployment guide. PR is WIP

### CODE REVIEW

File: deploy/README.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+# OpenTDF Deployment Guide
+
+Helm charts, configurations and support images for an opinionated deployment of OpenTDF
+
+## Prerequisites
+1. Install [Docker](https://www.docker.com/)
+
+  - see https://docs.docker.com/get-docker/
+
```
Comment (arkavo-com) [inline deploy/README.md:9]:
```suggestion
```
Docker not needed

File: deploy/README.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+   kubectl create namespace istio-ingress
+   kubectl label namespace istio-ingress istio-injection=enabled
+   helm install istio-ingress istio/gateway -n istio-ingress
+   ```
+   Example for AWS EKS using AWS CERT ARN:
+   ```
+   helm install istio-ingress istio/gateway -n istio-ingress -f charts/aws-eks-example.yaml
+   ```
+## Deploy OpenTDF
+ 1. Set install namespace: ```export ns=default```
```
Comment (arkavo-com) [inline deploy/README.md:45]: default namespace usage is frowned upon

File: deploy/README_OpenShift.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
+## Deployment:
+### 1. Update chart dependencies:
+helm dependency update .
+### 2. Install / Upgrade Chart
+#### Set certificates folder env var:
+```shell
+export CERTS_ROOT=certs
+```
+#### Create X509 Secret for Keycloak Truststore (Install only)
+Example using CA Certs in the certs directory:
```
Comment (arkavo-com) [inline deploy/README_OpenShift.md:10]: add an instruction to update `deploy/charts/local_gencert.sh` with your DOMAIN_NAME

File: deploy/README.md (hunk 2 above)
Comment (cassandrabailey293) [reply deploy/README.md:45]: I'm not sure how to fix this.

File: deploy/README.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+ 1. Enable Service Mesh Sidecar injection: ```kubectl label namespace $ns istio-injection=enabled```
+ 1. :warning: **Local Deploy Only** : Gateway TLS/HTTPS: Generate TLS Certs and create secret
```
Comment (pflynn-virtru) [inline deploy/README.md:47]: should we remove this "Local Deploy Only"?

Comment (cassandrabailey293) [reply deploy/README.md:47]: yeah we can probably remove that entirely

**Review by pflynn-virtru — DISMISSED** [review]

**Review by pflynn-virtru — APPROVED** [review]

**Pull request merged**